libraries/Microsoft.Bot.Builder.AI.LUIS/LuisAdaptiveRecognizer.cs

```diff
@@ -108,6 +108,20 @@ public LuisAdaptiveRecognizer()
         [JsonProperty("logPersonalInformation")]
         public BoolExpression LogPersonalInformation { get; set; } = "=settings.runtimeSettings.telemetry.logPersonalInformation";
 
+        /// <summary>
+        /// Gets or sets the intents that are possible to surface from this recognizer.
+        /// </summary>
+        /// <value>List of <see cref="IntentDescription"/>.</value>
+        [JsonProperty("possibleIntents")]
+        public ArrayExpression<IntentDescription> PossibleIntents { get; set; }
+
+        /// <summary>
+        /// Gets or sets the entities that are possible to surface from this recognizer.
+        /// </summary>
+        /// <value>List of <see cref="EntityDescription"/>.</value>
+        [JsonProperty("possibleEntities")]
+        public ArrayExpression<EntityDescription> PossibleEntities { get; set; }
+
         /// <inheritdoc/>
```

Comment:
Do we need to break down to intents and entities?  
Why not just an array of ids?

Reply:
And the ids appear in intents or entities when you do recognition so if you want to find them you have to look in both places.  Why the allergy to intents/entities?